templates/community_post_list_page.hbs

```diff
@@ -36,7 +36,7 @@
         </span>
       </span>
     <span class="post-to-community">
-      {{link 'new_post' role='button' class='button-large'}}
+      {{link 'new_post' role='button' class='button button-large'}}
```

Comment:
should these links actually have a `role='button'` . They are links :)

Reply:
Good point, they're links!